package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "macpro-quickstart-serverless",
-  "type": "module",
```

Comment:
@aalousi can you please put this line back? if not, why did we need to remove it?  Thanks!
When you re-add, please change `commonjs` back to `ES2020` in tsconfig.json.